src/InstallationValidator.Core/Reporting/ValueComparisonResultTeXBuilder.cs

```diff
@@ -1,14 +1,22 @@
 using System;
 using InstallationValidator.Core.Domain;
+using OSPSuite.Core.Domain;
 using OSPSuite.Infrastructure.Reporting;
 
 namespace InstallationValidator.Core.Reporting
 {
-   public abstract class ValueComparisonResultTeXBuilder<T> : OSPSuiteTeXBuilder<T> where T:  ValueComparisonResult
+   public abstract class ValueComparisonResultTeXBuilder<T> : OSPSuiteTeXBuilder<T> where T : ValueComparisonResult
    {
-      protected static string DeviationFor(T outputToReport)
+      private readonly DoubleFormatter _doubleFormatter;
+
+      protected ValueComparisonResultTeXBuilder()
+      {
+         _doubleFormatter = new DoubleFormatter();
+      }
+
+      protected string DeviationFor(T outputToReport)
       {
-         return $"{Assets.Reporting.Deviation}: {outputToReport.Deviation:0.####}{Environment.NewLine}";
```

Comment:
I am not sure where we have some double formatting but that should always be done with the DoubleFormatter